src/device-registry/routes/v2/airqlouds.js

```diff
@@ -47,14 +52,31 @@ const AdminLevelModel = (tenant) => {
 };
 
 const validAdminLevels = async () => {
-  const levels = await AdminLevelModel("airqo").distinct("name");
-  return levels.map((level) => level.toLowerCase());
+  try {
```

Comment:
@wabinyai and @BenjaminSsempala ....we just needed to add some error handling to get the actual errors in real time.